### PR TITLE
Tabular Stopping Metric + File Path Input

### PR DESCRIPTION
**Pull request by Innixma**
*Issue #, if available:*

*Description of changes:*

Major Changes:
- Added stopping_metric param to TabularPrediction
- stopping_metric changed to 'log_loss' by default when eval_metric is 'roc_auc'
- Added option for str type input to TabularPrediction train_data and tuning_data parameters

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-255-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-255/1/index.html

File: autogluon/task/tabular_prediction/tabular_prediction.py [hunk 1]
```diff
@@ -106,6 +99,11 @@ def fit(train_data, label, tuning_data=None, output_directory=None, problem_type
             For more information on these options, see `sklearn.metrics`: https://scikit-learn.org/stable/modules/classes.html#sklearn-metrics-metrics   
             
             You can also pass your own evaluation function here as long as it follows formatting of the functions defined in `autogluon/utils/tabular/metrics/`.
+        stopping_metric : function or str, default = None
+            Metric by which models use to early stop to avoid overfitting.
```
Comment (jwmueller) [inline autogluon/task/tabular_prediction/tabular_prediction.py]: typo: by which -> which

Comment (Innixma) [reply autogluon/task/tabular_prediction/tabular_prediction.py]: fixed

**Comment by szha** [comment]
Job PR-255-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-255/2/index.html

**Comment by szha** [comment]
Job PR-255-3 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-255/3/index.html
